.ci/generate_operators_doc.py

```diff
@@ -0,0 +1,163 @@
+
+
+def initialize_server(ansys_path=None, include_composites=False, include_sound=False):
+    server = dpf.start_local_server(ansys_path=ansys_path)
+    print(f"Ansys Path: {server.ansys_path}")
+    print(f"Server Info: {server.info}")
+    print(f"Server Context: {server.context}")
+    print(f"Server Config: {server.config}")
+    print(f"Server version: {dpf.global_server().version}")
+    if include_composites:
+        print("Loading Composites Plugin")
+        load_library(
+            Path(server.ansys_path)
+            / "dpf"
+            / "plugins"
+            / "dpf_composites"
+            / "composite_operators.dll"
+        )
+    if include_sound:
+        print("Loading Acoustics Plugin")
+        load_library(Path(server.ansys_path) / "Acoustics" / "SAS" / "ads" / "dpf_sound.dll")
+    return server
+
+
+def fetch_doc_info(server, operator_name):
+    spec = dpf.Operator.operator_specification(op_name=operator_name, server=server)
+    input_info = []
+    output_info = []
+    configurations_info = []
+    for input_pin in spec.inputs:
+        input = spec.inputs[input_pin]
+        input_info.append(
+            {
+                "pin_number": input_pin,
+                "name": input.name,
+                "types": [str(t) for t in input._type_names],
+                "document": input.document,
+                "optional": input.optional,
+            }
+        )
+    for output_pin in spec.outputs:
+        output = spec.outputs[output_pin]
+        output_info.append(
+            {
+                "pin_number": output_pin,
+                "name": output.name,
+                "types": [str(t) for t in output._type_names],
+                "document": output.document,
+                "optional": output.optional,
+            }
+        )
+    try:
+        for configuration_key in spec.config_specification:
+            configuration = spec.config_specification[configuration_key]
+            configurations_info.append(
+                {
+                    "name": configuration.name,
+                    "types": [str(t) for t in configuration.type_names],
+                    "document": configuration.document,
+                    "default_value": configuration.default_value_str,
+                }
+            )
+    except:
+        print(f"Operator {operator_name} - error in fetching configuration info!")
+    properties = spec.properties
+    plugin = properties.pop("plugin", "N/A")
+
+    category = properties.pop("category", None)
+
+    scripting_name = properties.pop("scripting_name", None)
+    if category and scripting_name:
+        full_name = category + "." + scripting_name
+    else:
+        full_name = None
+
+    user_name = properties.pop("user_name", operator_name)
+
+    op_friendly_name = user_name
+    if category:
+        op_friendly_name = category + ":" + op_friendly_name
+
+    license = properties.pop("license", "None")
+
+    exposure = properties.pop("exposure", "N/A")
```

Comment:
Hi @ansMHanmer here I would consider operators without exposure to be private. Operators without specs should for example be considered private.
```suggestion
    exposure = properties.pop("exposure", "private")
```